R/wwinference.R

```diff
@@ -160,6 +157,12 @@ wwinference <- function(ww_data,
     )
   }
 
+  fit_opts_use <- get_mcmc_options() # get defaults
+  # this overwrites all and only the values the user sets in `fit_opts`
+  fit_opts_use[names(fit_opts)] <- fit_opts
+
+  # maybe check for invalid options here?
```

Comment:
can use `formalArguments(compiled_model$sample)` or `formals(compiled_model$sample)` to get arguments that  `compiled_model$sample()` expects (with `formalArguments`) or the argument _and_ their default values (with `formals(compiled_model$sample)`).